src/core/public/core_app/status/lib/load_status.ts

```diff
@@ -96,6 +99,7 @@ function formatStatus(id: string, status: ServiceStatus): FormattedStatus {
 
   return {
     id,
+    original: status,
```

Comment:
Naming is hard, good job keeping context here!

Reply:
Not gonna lie, had a hard time between `state`, `status`, `status.state` and so on...